docker-app/qfieldcloud/core/views/projects_views.py

```diff
@@ -102,7 +102,7 @@ class ProjectViewSet(viewsets.ModelViewSet):
     serializer_class = ProjectSerializer
     lookup_url_kwarg = "projectid"
     permission_classes = [permissions.IsAuthenticated, ProjectViewSetPermissions]
-    pagination_class = pagination.PaginateResults
+    pagination_class = pagination.QfcLimitOffsetPagination(default_limit=25)
```

Comment:
Nice we can control the `default_limit` via an argument. Is it possible to setup a value globally in `settings.py` too? Something like `QFIELDCLOUD_API_DEFAULT_PAGE_LIMIT`?